src/cmd/build.js

```diff
@@ -53,10 +54,10 @@ export async function getDefaultLocalizedName(
   let extensionName: string = manifestData.name;
 
   try {
-    messageData = JSON.parse(await fs.readFile(messageFile));
+    messageData = parseJson(await fs.readFile(messageFile));
```

Comment:
This line can raise error for two different reasons:

- the `await fs.readFile(...)` part rejects (e.g. because the file doesn't exist or it cannot be read because of file permissions issues)
- the `parseJSON(...)` part rejects (e.g. because the file is not valid JSON)

In `src/util/manifest.js` we currently handle this two exceptions separately:

```js
  let manifestContents;

  try {
    manifestContents = await fs.readFile(manifestFile);
  } catch (error) {
    throw new InvalidManifest(
      `Could not read manifest.json file at ${manifestFile}: ${error}`);
  }

  let manifestData;

  try {
    manifestData = parseJSON(manifestContents, messageFile);
  } catch (error) {
    throw new InvalidManifest(
      `Error parsing manifest.json at ${manifestFile}: ${error}`);
  }
```

we should do something similar here (but with `UsageError` instead of `InvalidManifest`)